packages/wonder-blocks-clickable/components/clickable.js

```diff
@@ -217,10 +252,9 @@ export default class Clickable extends React.Component<Props> {
             >
                 {(state, handlers) =>
                     this.getCorrectTag(state, {
-                        // eslint-disable-next-line react/prop-types
-                        "aria-label": this.props["aria-label"],
```

Comment:
We can remove this as `...restProps` takes care of this?

Reply:
Remove the `data-test-id`?

Reply:
Why did you remove the aria-label? Is it safe to do so?

Reply:
Sorry... misparsed your question.  `AriaProps` includes `aria-label` so now that we're including `AriaProp` in the props for this component spreading `restProps` handles `aria-label` for us.